url_shortener/models.py

```diff
@@ -38,7 +38,7 @@ class Link(models.Model):
     1
 
     """
-    url = models.URLField(verify_exists=False)
+    url = models.URLField()
```

Comment:
`verify_exists` removed in 1.5: https://docs.djangoproject.com/en/dev/internals/deprecation/#deprecation-removed-in-1-5

> django.db.models.fields.URLField.verify_exists will be removed. The feature was deprecated in 1.3.1 due to intractable security and performance issues and will follow a slightly accelerated deprecation timeframe.